core/controllers/review_tests_test.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the reivew tests page."""
+
+from constants import constants
+from core.domain import story_domain
+from core.domain import story_services
+from core.domain import user_services
+from core.tests import test_utils
+import feconf
+
+
+class BaseReviewTestsControllerTests(test_utils.GenericTestBase):
+
+    OWNER_EMAIL = 'owner@example.com'
+    VIEWER_EMAIL = 'viewer@example.com'
+    VIEWER_USERNAME = 'viewer'
+
+    def setUp(self):
+        """Completes the sign-up process for the various users."""
+        super(BaseReviewTestsControllerTests, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+
+        self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
+
+        self.set_admins([self.ADMIN_USERNAME])
+        self.admin = user_services.UserActionsInfo(self.admin_id)
+
+        self.story_id_1 = 'story_id_1'
+        self.story_id_2 = 'story_id_2'
+        self.node_id = 'node_1'
+        self.node_id_2 = 'node_2'
+        self.exp_id = 'exp_id'
+
+        self.save_new_valid_exploration(self.exp_id, self.owner_id)
+        self.publish_exploration(self.owner_id, self.exp_id)
+
+        self.node_1 = {
+            'id': self.node_id,
+            'title': 'Title 1',
+            'destination_node_ids': [],
+            'acquired_skill_ids': ['skill_id_1', 'skill_id_2'],
+            'prerequisite_skill_ids': [],
+            'outline': '',
+            'outline_is_finalized': False,
+            'exploration_id': self.exp_id
+        }
+
+        self.story = story_domain.Story.create_default_story(
+            self.story_id_1, 'Public Story Title')
```

Comment:
@sophiewu6, `story_domain.Story.create_default_story` method is changed recently, so you'll have to add new parameter corresponding to topic id here. I think the tests are failing because of this.   

(Check #6855 for more info.)

Reply:
Thank you for pointing that out! I have fixed it.